document-generation/utils/sampleTestData.ts

```diff
@@ -1601,9 +1603,9 @@ export const sampleJustificationAndApproval = {
     },
     scope: "J & A Scope Test 1",
     contractingShop: "DITCO",
-    purchaseRequestNumber: "OYYMM-000-000-000000, ",
-    agencyLabel: "Business Transformation Agency (BTA)",
-    taskOrderType: "Firm Fixed Price",
+    purchaseRequestNumber: "OYYMM-000-000-000000",
+    agencyLabel: "Business Transformation Agency (BTA)/Jeff Org J&A",
+    taskOrderType: "firm-fixed-price",
```

Comment:
I find this surprising. Why is this value kebab case?

Reply:
DITCO preference